### PR TITLE
Add basic DOAP file for Pulsar

### PR DESCRIPTION
**Pull request by eolivelli**
This patch adds a default DOAP file for Pulsar

See here for reference
https://projects.apache.org/create.html

### CODE REVIEW

**Comment by eolivelli** [comment]
@Anonymitaet @Jennifer88huang  @dave2wave 
once we commit this file I expect it to be available at https://pulsar.apache.org/doap.rdf

and then I have to a reference to that link here:
https://svn.apache.org/repos/asf/comdev/projects.apache.org/trunk/data/projects.xml

File: site2/website/static/doap.rdf [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+<?xml-stylesheet type="text/xsl"?>
+<rdf:RDF xml:lang="en"
+         xmlns="http://usefulinc.com/ns/doap#" 
+         xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" 
+         xmlns:asfext="http://projects.apache.org/ns/asfext#"
+         xmlns:foaf="http://xmlns.com/foaf/0.1/">
+<!--
+  =======================================================================
+
+   Copyright (c) 2011 The Apache Software Foundation.
```
Comment (merlimat) [inline site2/website/static/doap.rdf:11]: This should probably be 2021 and maybe we can use the regular license header here too

File: site2/website/static/doap.rdf [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<!--
+  =======================================================================
+
+   Copyright (c) 2011 The Apache Software Foundation.  
+   All rights reserved.
+
+  =======================================================================
+-->
+  <Project rdf:about="https://pulsar.apache.org/">
+    <created>2011-02-06</created>
```
Comment (merlimat) [inline site2/website/static/doap.rdf:17]: We should use `2018-09-19`, when the TLP was established.

File: site2/website/static/doap.rdf [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      </GitRepository>
+    </repository>
+    
+    <maintainer>
+      <foaf:Person>
+        <foaf:name>Apache Pulsar Community</foaf:name>
+        <foaf:mbox rdf:resource="mailto:dev@pulsar.apache.org"/>
+      </foaf:Person>
+    </maintainer>
+
```
Comment (dave2wave) [inline site2/website/static/doap.rdf:43]: Should include a list of Pulsar releases.

Here is a page with a template - https://projects.apache.org/create.html

Once the DOAP file is created here are instructions for connecting it to projects.apache.org - https://projects.apache.org/about.html

File: site2/website/static/doap.rdf [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  =======================================================================
+-->
+  <Project rdf:about="https://pulsar.apache.org/">
+    <created>2011-02-06</created>
+    <license rdf:resource="http://usefulinc.com/doap/licenses/asl20" />
+    <name>Apache Pulsar</name>
+    <homepage rdf:resource="https://pulsar.apache.org/" />
+    <asfext:pmc rdf:resource="https://pulsar.apache.org" />
+    <shortdesc>A distributed pub-sub messaging system.</shortdesc>
+    <description>Pulsar is a distributed pub-sub messaging platform with a very flexible messaging model and an intuitive client API..</description>
```
Comment (Anonymitaet) [inline site2/website/static/doap.rdf:23]:
```suggestion
    <description>Pulsar is a distributed pub-sub messaging platform with a very flexible messaging model and an intuitive client API.</description>
```

File: site2/website/static/doap.rdf (hunk 3 above)
Comment (eolivelli) [reply site2/website/static/doap.rdf:57]: @dave2wave I tried to use the sample XML generator but it creates a simpler file.

Also I cannot understand what you mean by "Should include a list of Pulsar releases"

Usually releases are recorded using the standard "reporter.apache.org" tool that is not related to the DOAP file

**Review by eolivelli — COMMENTED** [review]
@merlimat @dave2wave PTAL I have updated the patch

**Review by dave2wave — APPROVED** [review]
This will be good for now

**Pull request merged**